### PR TITLE
[Edit] update to blur.md added image example

### PR DESCRIPTION
**Pull request by DanWSDev**
### Description
Added new png image to show the example

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/filter-functions/terms/blur/blur.md [hunk 1]
```diff
@@ -33,3 +33,5 @@ Give the image a blur radius of 4px:
   filter: blur(4px);
 }
 ```
+
+![css-filterfunctions-blur-example](https://github.com/DanWSDev/docs/blob/patch-3/media/css-filterfunctions-blur-example.png)
```
Comment (caupolicandiaz) [inline content/css/concepts/filter-functions/terms/blur/blur.md:37]: the image should be in the docs/media folder and the link should use the following path -
https://raw.githubusercontent.com/Codecademy/docs/main/media/css-filterfunctions-blur-example.png

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@DanWSDev Just one edit. Please update, and tag me when complete. Thanks.

**Comment by DanWSDev** [comment]
> @DanWSDev Just one edit. Please update, and tag me when complete. Thanks.

@caupolicandiaz Thank you for pointing this out silly mistake on my part it is updated now.

**Review by caupolicandiaz — APPROVED** [review]
@DanWSDev Looks good, this is ready for a second review

**Review by SSwiniarski — APPROVED** [review]
@DanWSDev Looks good.

**Pull request merged**